sdk/core/azure-core/inc/azure/core/version.hpp

```diff
@@ -32,7 +34,7 @@ namespace Azure { namespace Core { namespace Details {
     static constexpr int Patch = AZURE_CORE_VERSION_PATCH;
 
     /// Optional pre-release identifier. SDK is in a pre-release state when not empty.
-    static std::string const PreRelease;
+    AZ_CORE_EXPORT static std::string const PreRelease;
```

Comment:
Once merged, we should add this in the guidelines about statics and update our extensible enum section:
https://azure.github.io/azure-sdk/cpp_implementation.html#enumerations

cc @CaseyCarter